test/gtest/uct/ib/test_rc.cc

```diff
@@ -391,6 +391,22 @@ UCS_TEST_SKIP_COND_P(test_rc_get_limit, get_zcopy_purge,
     post_max_reads(m_e1, sendbuf, recvbuf);
 
     scoped_log_handler hide_warn(hide_warns_logger);
+
+    unsigned flags      = UCT_FLUSH_FLAG_CANCEL;
+    ucs_time_t deadline = ucs::get_deadline();
+    ucs_status_t status;
+    do {
+        ASSERT_EQ(1ul, m_e1->num_eps());
+        status = uct_ep_flush(m_e1->ep(0), flags, NULL);
+        progress();
+        if (flags & UCT_FLUSH_FLAG_CANCEL) {
```

Comment:
will this new flow trigger `uct_rc_txqp_purge_outstanding` while having outstanding (uncompleted) get operations  somehow?

Reply:
yes, from here for example
https://github.com/openucx/ucx/blob/5c71925431166d43357a3ed1aee4fc378cb81a16/src/uct/ib/rc/accel/rc_mlx5_ep.c#L537